client/settings/payment-methods-list/payment-method.tsx

```diff
@@ -34,96 +32,40 @@ import Jcb from 'assets/images/payment-method-icons/jcb.svg?asset';
 import Cartebancaire from 'assets/images/cards/cartes_bancaires.svg?asset';
 import UnionPay from 'assets/images/cards/unionpay.svg?asset';
 import PAYMENT_METHOD_IDS from 'wcpay/constants/payment-method';
+import usePaymentMethodAvailability from 'wcpay/settings/use-payment-method-availability';
+import InlineNotice from 'wcpay/components/inline-notice';
+import { useEnabledPaymentMethodIds } from 'wcpay/data';
 
 interface PaymentMethodProps {
 	id: string;
 	label: string;
 	// eslint-disable-next-line @typescript-eslint/naming-convention
 	Icon: () => JSX.Element | null;
 	description: string;
-	status: string;
-	checked: boolean;
 	onCheckClick: ( id: string ) => void;
 	onUncheckClick: ( id: string ) => void;
 	className?: string;
-	isAllowingManualCapture: boolean;
-	isSetupRequired?: boolean;
-	setupTooltip?: string;
-	required: boolean;
 	locked: boolean;
-	isPoEnabled: boolean;
-	isPoComplete: boolean;
 }
 
-const documentationTypeMap = {
-	DEFAULT:
-		'https://woocommerce.com/document/woopayments/payment-methods/additional-payment-methods/#method-cant-be-enabled',
-	BNPLS:
-		'https://woocommerce.com/document/woopayments/payment-methods/buy-now-pay-later/#contact-support',
-};
-
-const getDocumentationUrlForDisabledPaymentMethod = (
-	paymentMethodId: string
-): string => {
-	const paymentMethodConfig =
-		window.wooPaymentsPaymentMethodsConfig?.[ paymentMethodId ];
-
-	if ( paymentMethodConfig?.isBnpl ) {
-		return documentationTypeMap.BNPLS;
-	}
-
-	return documentationTypeMap.DEFAULT;
-};
-
 const PaymentMethodLabel = ( {
+	id,
 	label,
-	required,
-	status,
-	disabled,
 }: {
+	id: string;
 	label: string;
-	required: boolean;
-	status: string;
-	disabled: boolean;
 } ): React.ReactElement => {
+	const { chip, chipType = 'warning' } = usePaymentMethodAvailability( id );
+
 	return (
 		<>
 			{ label }
-			{ required && (
+			{ PAYMENT_METHOD_IDS.CARD === id && (
 				<span className="payment-method__required-label">
 					{ '(' + __( 'Required', 'woocommerce-payments' ) + ')' }
 				</span>
 			) }
-			{ upeCapabilityStatuses.PENDING_APPROVAL === status && (
```

Comment:
All of this has been moved to `client/settings/use-payment-method-availability/index.tsx`